docker-compose-files/openshift-compose/README.md

```diff
@@ -0,0 +1,48 @@
+### OpenShift import docker-compose
+
+
+Start ADB/CDK
+
+To get latest openshift edit Vagrantfile make `IMAGE_TAG` to `IMAGE_TAG="latest"`
+```bash
+vagrant up && vagrant provision
```

Comment:
This step assumes you have Vagrantfile required for running ADB/CDK available with you.